### ISSUE TITLE
Missing support for ARM processors

### ISSUE THREAD

**Issue by cwinters8** [newrelic/newrelic-cli#774]
TLDR: Are there any plans to support ARM based processors, such as the new-ish AWS Graviton 2 processors? 

I specifically want to be able to monitor my MongoDB infrastructure using New Relic, but it seems there isn't any support for ARM in either the CLI or in the New Relic Infra MongoDB integration. It seems like my options are either change instance types (don't want to do - the m6g.medium EC2 instance type is just right for our use case currently, and going to an m5 instance means significantly higher expense) or switch to DataDog (would prefer to use NR). Please let me know if I'm missing an option. Thanks in advance.

### Description
Cannot install on ARM based Amazon Linux instances

### Current behavior
Attempting to install using the following command:
```bash
curl -Ls https://raw.githubusercontent.com/newrelic/newrelic-cli/master/scripts/install.sh | bash && sudo NEW_RELIC_API_KEY=REDACTED NEW_RELIC_ACCOUNT_ID=3112141 /usr/local/bin/newrelic install -n mongodb-open-source-integration
```
Results in:
```
Starting installation.
Installing New Relic CLI v0.20.3
Installing to /usr/local/bin using sudo
/usr/local/bin/newrelic: /usr/local/bin/newrelic: cannot execute binary file
```
Based on some additional investigation in the releases for newrelic-cli, it looks like only x86 architectures are supported currently.

### Expected behavior
I would expect the CLI to be able to execute after installation.

**Comment by jthurman42** [comment]
There shouldn't be anything specific in the CLI that breaks on ARM, this is likely more of an issue that the build system does not generate an artifact (currently only `amd64` is build: https://github.com/newrelic/newrelic-cli/blob/master/.goreleaser.yml#L21)

Each integration on the other hand would need to be tested / build / updated as well.

**Comment by jthurman42** [comment]
Tested goreleaser with the added `arm64` field, and there were no compilation issues.  I did not spin up an arm64 host to test the binary after the fact.  ( https://github.com/newrelic/newrelic-cli/blob/master/.goreleaser.yml#L21 )

Also note that the install script above would need ARCH detection, it's hard-coded at `X86_64`: https://github.com/newrelic/newrelic-cli/blob/master/scripts/install.sh#L50

**Comment by cwinters8** [comment]
Awesome, thank you @jthurman42. I will give that a shot.

Also, look what appeared today for nri-mongodb!! I'm so happy!
![image](https://user-images.githubusercontent.com/40615752/112361257-331b9100-8ca1-11eb-90a3-3d41f75998a4.png)
